lib/otel/span-processor.js

```diff
@@ -54,7 +62,7 @@ module.exports = class NrSpanProcessor {
       let sanitized = value
       if (key === SEMATTRS_NET_PEER_PORT) {
         key = 'port_path_or_id'
-      } else if (prop === SEMATTRS_NET_PEER_NAME) {
+      } else if (prop === SEMATTRS_NET_PEER_NAME || prop === ATTR_SERVER_ADDRESS || prop === SEMATTRS_HTTP_HOST) {
```

Comment:
i've refactored this to split between relevant span/segments but this is fine for now.